.travis.yml

```diff
@@ -2,7 +2,9 @@ language: ruby
 script: "bundle exec rake"
 rvm:
   - 1.9.3
-  - 2.0.0
+  - 2.0.-
+  - 2.1.0
+  - 2.2.0
```

Comment:
Ruby 2.2 is failing because of a Sqlite compilation error. I fixed this by removing the Sqlite version constraint in my PR: https://github.com/nathanvda/cocoon/pull/352/files#r55928263